cs-amazon/src/main/java/io/cloudslang/content/amazon/utils/OutputsUtil.java

```diff
@@ -33,4 +52,31 @@ private OutputsUtil() {
 
         return signatureReturnResultMap;
     }
+
+    public static void putResponseIn(Map<String, String> queryMapResult) {
+        if (queryMapResult != null) {
```

Comment:
@mihaitusa I don't really like this, I think it's code duplication, I prefere this version. But if you want i can refactor this. What do you think?